src/code_style.md

```diff
@@ -0,0 +1,37 @@
+# Clear Code  
+
+1. Code should be clear and concise.  
+2. Variables in the code should have meaningful names.  
+3. Function naming should follow established theories and naming conventions.  
```

Comment:
you could link to either a code style or the julia manual, or define the naming convention. Also I feel like "established theories" sounds too scientific, likely there are no actual theories about naming things?